Messenger/src/test/java/com/epam/testmodule/ChoseModeTest.java

```diff
@@ -41,7 +41,7 @@ public void CheckInput_whenInputEqualsTwo_ShouldRunFileModerRun() throws IOExcep
         FileMode fileMode = Mockito.spy(new FileMode());
         ChoseMode choseMode = new ChoseMode(consoleMode,fileMode);
         doReturn("C:\\Users\\Ivan_Pylypiv\\Desktop\\Library\\Messenger\\data2.txt").when(fileMode).getInfoFromUser();
-        doReturn(true).when(fileMode).checkFilePath("C:\\Users\\Ivan_Pylypiv\\Desktop\\Library\\Messenger\\data2.txt");
+        doReturn(true).when(fileMode).checkFilePath("C:\\Users\\Ivan_Pylypiv\\IdeaProjects\\Library\\Messenger\\data2.txt");
```

Comment:
pls avoid using absolute file paths in the source code, as this project may be running on a different machine